joint_trajectory_controller/test/test_trajectory_controller.cpp

```diff
@@ -95,6 +95,87 @@ TEST_P(TrajectoryControllerTestParameterized, configure_state_ignores_commands)
   executor.cancel();
 }
 
+TEST_P(TrajectoryControllerTestParameterized, check_interface_names)
+{
+  SetUpTrajectoryController();
+
+  const auto state = traj_controller_->get_node()->configure();
```

Comment:
I am seeing this info message which does not see right.
```
[INFO] [1660165165.826510454] [test_joint_trajectory_controller]: No specific joint names are used for command interfaces. Using 'joints' parameter.
```
I think this needs to be fixed.

Reply:
Oh I see, this is intended. The  next test is testing the behavior where command_joints is set. Great!